analyticConfigurations/metric_meta-java.jvm.json

```diff
@@ -47,6 +47,24 @@
         },
         "validMax" : 100
       }
+    }, {
+      "match" : "netuitive.jvm.non-heap.utilizationpercent",
```

Comment:
Are there any others in this family? Could you get away with `netuitive.jvm.*.utilizationpercent`?

Reply:
We absolutely could do that. There are others in the family each with the same setting so it would make more sense to do it this way.

Reply:
This way as in explicitly setting the config for each metric?

Reply:
This way as in the way you suggested. Sorry I wasn't clear.